src/utils/filterTodos.ts

```diff
@@ -0,0 +1,22 @@
+import { Todo } from '../types/Todo';
+import { Status } from '../types/Status';
+
+export const filterTodos = (
+  todos: Todo[],
+  query: string,
+  status: Status,
+): Todo[] => {
+  return todos
+    .filter(todo => {
```

Comment:
- if there are no query and status is all you don't need to filter at all
- also, you can try to do it in one filter